### PR TITLE
Add "Deploy from Docker Image Option" to Quickstart

### PR DESCRIPTION
**Pull request by unicodeveloper**
Fulfills Linear Issue #LOG-2571.

Add a third option for deployment from a Docker Image to the Quickstart Tutorial.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "contentlayer": "^0.3.4",
     "cookies-next": "^4.2.1",
     "copy-to-clipboard": "^3.3.1",
+    "date-fns": "^4.1.0",
```
Comment (coffee-cup) [inline package.json:29]: Is `date-fns` necessary if we also have `dayjs`?

File: src/docs/quick-start.md [hunk 2]
```diff
@@ -90,11 +92,66 @@ width={836} height={214} quality={100} />
...
+width={836} height={714} quality={100} />
+
+After the project is created, you will land on the <a href="/quick-start#the-canvas" target="_blank">Project Canvas</a>. A panel will appear prompting you to Add a Service.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1727281215/docs/quick-start/add_a_service.png"
+alt="screenshot of add a service panel on the canvas"
+layout="responsive"
+width={1422} height={1284} quality={100} />
+
+- Click **Add a Service** and select **Docker Image** option from the modal that pops up.
```
Comment (brody192) [inline src/docs/quick-start.md:121]: Perhaps - "and select the **Docker Image** option"

I think the addition of "the" makes the sentence flow better.

File: package.json (hunk 1 above)
Comment (brody192) [reply package.json:29]: jr beat me to asking what it was for haha

File: src/docs/quick-start.md (hunk 2 above)
Comment (unicodeveloper) [reply src/docs/quick-start.md:121]: I also agree. Had it there but when I ran it through a grammar correction service, it suggested removing "the". So weird 😆

Comment (unicodeveloper) [reply src/docs/quick-start.md:121]: Adding it. It makes it flow better like you mentioned.

**Comment by brody192** [comment]
I Have no clue why the build is failing, that code hasn't been touched in a while.

Other than that, this looks good to me!

**Comment by unicodeveloper** [comment]
> I Have no clue why the build is failing, that code hasn't been touched in a while.
> 
> Other than that, this looks good to me!

I am pulling my hair here trying to figure out why it is failing. Don't know if its a typescript or Nextjs problem 🤦🏿

**Comment by brody192** [comment]
I pulled your comment locally, and I'm not having any issues building, the joys of the node ecosystem!

**Comment by unicodeveloper** [comment]
> I pulled your branch locally, and I'm not having any issues building, the joys of the node ecosystem!

Same here. I just noticed that in my editor, it raises a Typescript error and this is the exact error that is causing the build to fail on the CI even though it runs locally without any issues. @brody192 

CleanShot 2024-09-25 at 19 43 28@2x

**Comment by brody192** [comment]
Why am I not getting any TS errors in my vscode, this is odd.

**Comment by unicodeveloper** [comment]
> Why am I not getting any TS errors in my vscode, this is odd.

I'm going to prolly come check it out again later. @coffee-cup please help if you have any ideas on fixing this 🙏🏿

**Comment by brody192** [comment]
We are so back!

**Comment by unicodeveloper** [comment]
> We are so back!

Wow! Was it your update of the yarn.lock file that fixed it? @brody192

**Comment by brody192** [comment]
Yeah, going forward its best to install deps locally with `yarn install --frozen-lockfile` so the lockfile doesn't get updated.

**Comment by unicodeveloper** [comment]
Ah, that makes sense. Thanks for the tip!

*Best Regards,*

*Prosper Otemuyiwa*
*Staff Developer Advocate*
 @unicodeveloper <https://twitter.com/unicodeveloper> | GitHub | Twitter

**Comment by brody192** [comment]
Well, looks good to me, feel free to merge!

**Pull request merged**